### PR TITLE
feat: umbrel connector

### PR DESCRIPTION
**Pull request by reneaaron**
TODO: 

- [x] Find a proper icon
- [x] Figure out how to change the connect URI to `http://umbrel.local:58000/apps/new?...`
- [x] Add intermediate step for description how to connect (otherwise the popup just opens a non-existent page :thinking:)

### CODE REVIEW

File: src/components/bc-connector-list.ts [hunk 1]
```diff
@@ -26,7 +27,7 @@ export class ConnectorList extends withTwind()(BitcoinConnectElement) {
       connectors.push(html`<bc-lnc-connector></bc-lnc-connector>`);
     }
 
-    return html` <div class="flex justify-center items-start flex-wrap gap-4">
```
Comment (reneaaron) [inline src/components/bc-connector-list.ts:29]: A bit more room to breathe :+1:

File: src/components/connectors/ConnectorElement.ts [hunk 2]
```diff
@@ -29,17 +29,17 @@ export abstract class ConnectorElement extends withTwind()(
...
       <div
-        class="w-16 h-16 drop-shadow-lg rounded-2xl flex justify-center items-center overflow-hidden"
+        class="w-16 h-16 drop-shadow rounded-2xl flex justify-center items-center overflow-hidden"
         style="background: ${this._background};"
       >
         ${this._icon}
       </div>
       <span
-        class="text-sm font-sans font-medium ${classes[
+        class="mt-2 text-sm font-sans font-medium ${classes[
```
Comment (reneaaron) [inline src/components/connectors/ConnectorElement.ts:42]: Put connector names closer to their icon.

File: src/components/pages/bc-umbrel.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    await store.getState().connect({
+      nwcUrl: nwc.getNostrWalletConnectUrl(),
+      connectorName: 'Umbrel',
+      connectorType: 'nwc.umbrel',
+    });
+    if (!store.getState().connected) {
+      store.getState().setRoute('/nwc');
+      // TODO: show an error message directly on this page
+      alert('Failed to connect. Please check your NWC URL');
+    }
```
Comment (reneaaron) [inline src/components/pages/bc-umbrel.ts:52]: Feels like quite a hack, doesn't it? Any ideas how we could simplify this?

**Review by rolznz — APPROVED** [review]
tACK, nice! and thanks for the additional minor fixes.

File: src/components/pages/bc-umbrel.ts (hunk 3 above)
Comment (rolznz) [reply src/components/pages/bc-umbrel.ts:52]: Yes, I think we can extract the skeleton loading code into a new component and render that from each page instead.

But each page will have a loading state and error messages - I think there will be quite a lot of duplication. I wonder if there is a way to simplify it as a follow up step.

I plan to address this in a separate PR. I made an issue here: https://github.com/getAlby/bitcoin-connect/issues/83

**Comment by rolznz** [comment]
@reneaaron I am happy to merge it, can you change it out of draft status?

File: src/components/pages/bc-umbrel.ts (hunk 3 above)
Comment (reneaaron) [reply src/components/pages/bc-umbrel.ts:52]: Okay, cool! :sunglasses:

**Pull request merged**